README.md

```diff
@@ -57,7 +57,7 @@ chpwd () {
 
   if [ -f $FILE ];
   then
-     nodengine
+     installedVersion=$(nodengine | tail -n 1) && [ -n "$installedVersion" ] && nvm use $installedVersion
```

Comment:
this documentation is very closest with the nvm problem. Write this in a separate user case